.pfnci/wheel-linux/main.sh

```diff
@@ -13,6 +13,13 @@ if [ -z "${BRANCH}" ]; then
     BRANCH="$(cat ./.pfnci/BRANCH)"
 fi
 
+# TODO: revert after confirming CUDA 11.6 driver work on GCP
+if [[ "${CUDA}" = "11.6" ]]; then
+    export CUPY_RELEASE_SKIP_VERIFY=1
+else
```

Comment:
Ah, we should overriding the value set in `config.pbtxt` (for ROCm)

```suggestion
elif [[ "${CUPY_RELEASE_SKIP_VERIFY:-}" != "" ]]; then
```

Reply:
thoughtful